src/main.rs

```diff
@@ -63,13 +51,7 @@ async fn main() {
                 println!("Git repo initialized.");
             }
             if Config::load().is_err() {
-                let contents = format!(
-                    "# This is an example config
-# Adapt it to your needs and check if its valid via `website-stalker check`.
-# In order to run use `website-stalker run --all`.
-{}",
-                    Config::example_yaml_string()
-                );
+                let contents = config::EXAMPLE_CONF.to_string();
                 fs::write("website-stalker.yaml", contents)
```

Comment:
We are getting into nitpicks!

This can also be simplified. (Looks like I can’t create a multi line suggestion on the mobile web interface…)

```suggestion
                fs::write("website-stalker.yaml", EXAMPLE_CONF)
```